src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/QueryStringValidation.feature

```diff
@@ -0,0 +1,186 @@
+Feature: Query Strings handling for GET requests
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given there are no schools
+    When sending a GET request to "/ed-fi/schools?totalCount=true"  
+            """
+            """
+    Then there is a response header with "totalCount" equals to 0
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?totalCount=true" 
+            """
+            """
+    Then there is a response header with "totalCount: 2" 
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?totalCount=false" 
+            """
+            """
+    Then it should respond with 200
+    And header "totalCount" is not present
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
```

Comment:
In this file, it is not important that the school have multiple grade levels. It will be easier to read the file if we simplify line 39 to have a single grade instead of four.

Reply:
Done